autosave/mixins.py

```diff
@@ -168,7 +169,7 @@ def autosave_media(self, obj=None, get_params=''):
 
         return forms.Media(js=(
             reverse('admin:%s_%s_autosave_js' % info, args=[pk]) + get_params,
-            "%sautosave/js/autosave.js?v=2" % settings.STATIC_URL,
+            static('autosave/js/autosave.js'),
```

Comment:
You don't need to wrap the path string in anything in current versions of Django. `'autosave/js/autosave.js?v=2'` should work on its own. This is just some legacy stuff from when that was required. We're slowing taking it out of our codebase. 

That said, the `?v=2` is important because our CDN caches static files practically forever if we don't cachebust it.